.github/workflows/format.yml

```diff
@@ -0,0 +1,35 @@
+name: Format
+
+on: 'push'
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+          
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Format
```

Comment:
This is a manual tasks and it is also run by husky as a commit hook ;)

Reply:
Why was it then that when I ran `yarn format` it changed more than 20 files that haven't been touched in quite a while? :thinking: